tests/upgrade/externalService_test.js

```diff
@@ -62,14 +62,14 @@ Scenario(
 Scenario(
   'Verify Redis as external Service Works After Upgrade @post-external-upgrade @post-client-upgrade',
   async ({
-    I, grafanaAPI, remoteInstancesHelper,
+    I, grafanaAPI, remoteInstancesHelper, inventoryAPI,
```

Comment:
Please remove the `inventoryAPI`, if it's not used within the scenario.

Reply:
Removed.